BDArmory/Control/BDModulePilotAI.cs

```diff
@@ -942,6 +942,7 @@ public float MaxLiftAcceleration
 
         double commandSpeed;
         Vector3d commandHeading;
+        Vector3d lastCommandCoords = Vector3d.zero;
```

Comment:
It's not necessary to store the `lastCommandCoords`. If `ReleaseCommand(false)` is used, then `assignedPositionGeo` won't be updated from its previous coords.
I think a better approach here would be to store the previous command in `BDGenericAIBase` and then the check below becomes a check for the current command being `Free` and the previous one being `Attack`, at which point you'd just call `CommandAttack(assignedPositionGeo)`. `ReleaseCommand(true)` would then also need to set the previous command to be `Free`. This would give us options for resuming `FlyTo` and `Follow` commands too.

Reply:
This could be added as a `bool ResumeCommand()` option in `BDGenericAIBase` that acts as a no-op and returns `false` if the previous command is `Free` and otherwise resumes the previous command with the current `assignedPositionGeo` returning `true`.

Reply:
I've made the change I described, which should allow resuming various commands once the cause of the interruption is removed, e.g., Attacking a position, find something along the way and fight it, resume to the original Attack position. Can you try it and see what you think?

Reply:
Yes. will give it a try.

Reply:
Works nicely. Test plane broke off from an AttackMove command to engage a target enroute, then returned to the AttackMove; second test had MoveTo breaking off to evade incoming missile, then resumed MoveTo. Didn't test follow, but presumably it will function similarly.